src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        int figureNumber = 6;
+        Figure[] figures = new Figure[figureNumber];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+            figures[i].draw();
+        }
+        for (int x = figures.length / 2; x < figures.length; x++) {
+            figures[x] = figureSupplier.getDefaultFigure();
+            figures[x].draw();
+        }

```

Comment:
lets use just 1 for-loop

Reply:
I can replace for-loop with while-loop or write manually